### ISSUE TITLE
Anime cover image at sidebar

### ISSUE THREAD

**Issue by thekot** [erengy/taiga#237]
It would be great to have a selected anime cover image displayed at sidebar above "Now Playing". I don't really know is it hard to implement or not but I think the idea is good. As for me I often open "Anime Information" window just to see that image because I don't clearly remember all anime by titles.

Also it may work as shortcut to open anime web page.

![Mock](https://i.imgur.com/s3isQW1.png)

**Comment by erengy** [comment]
I'm not fond of this idea primarily because it breaks the logical structure of the UI (i.e. selecting an item inside Anime List page causes an image to be displayed outside of it). There are also other concerns such as the variable height of poster images, sidebar items moving up and down if the image is placed at the top, and so on.

I think that adding grid-view mode to Anime List (#87) is a much better idea overall, which satisfies the same use case.

**Comment by VivaLaPanda** [comment]
Agreed, this view mode is just awkward, but exploring additional UI designs is important. I think UI overhaul is listed as a long term goal.

Thanks,
Adrian Smith
adrianrsmith.com [adrianrsmith.com]
On 3/2/2016 17:35:22, Eren Okka <notifications@github.com> wrote:
I'm not fond of this idea primarily because it breaks the logical structure of the UI (i.e. selecting an item inside Anime List page causes an image to be displayed outside of it). There are also other concerns such as the variable height of poster images, sidebar items moving up and down if the image is placed at the top, and so on.
I think that adding grid-view mode to Anime List (#87 [https://github.com/erengy/taiga/issues/87]) is a much better idea overall, which satisfies the same use case.

**Comment by thekot** [comment]
In my opinion it does not breaks anything. Its like a preview pane it can't be inside of the list view. It could have static size to fight variable posters heights. It could hold it's place even if no applicable item are selected to prevent sidebar items jumpy behaviour or it could be moved to the bottom of the sidebar.

As for grid-view. The idea is different somehow. The grid-view will occupy large amount of space. I personally prefer to use details list view because  it's compact, contain more information and allows me to quickly sort items.

There are some examples how this kind of functionary implemented in different apps: [Windows Explorer](http://i.imgur.com/J822rVf.png) and [Dialog Boxes](http://i.imgur.com/5yun081.png), [TotalCommander](http://i.imgur.com/UyH3fXz.png), [FastStone Image Viewer](http://i.imgur.com/TabQdSS.png).

**Comment by amboris** [comment]
how about put it below the torrent feed icon. there is so much space on the left side bar and im so bad at names without pictures. can you pls add pic preview so i dont need to double klick for more info to see the cover pic of the anime?

**Comment by koteq** [comment]
I was able to implement the feature without any C++ knowledge and seeing VS2015 first time in my life.
I just thought it's could be easily implemented and it was so.
![Screenshot](https://i.imgur.com/HXLXzWw.png)

~I don't think that you interested in my shitty implementation so I won't share any. I'll be still waiting to see it in the v2.0. Thanks for great app.~

**Comment by amboris** [comment]
oh come on man don't be so mad. its a one man project and @erengy just dont have much time to work on it. even if he does there a so many feature requests to go through. the reason why most people put up their project on github is that other can help the project growing. 

btw this is what i need!

pls just go to code and make a pull request so he can implement. and all of us can be happy ;)
if you got the drive you could even go check out the other request and implement it through pull request and we will reach v2.... way faster than with one developer.

**Comment by koteq** [comment]
@amboris I won't be able to make the feature to work consistently in each section of the app nor write clean code which may be acceptable by project owner.

Now I think it was a bad idea to write about what I did, but I was really happy at the moment so couldn't resist. Sorry if I insulted someone, software development is a serious business.

**Comment by erengy** [comment]
Maybe I'm the one who misread the comment in question, but it didn't seem offensive to me. In fact, if you were able to do it without any prior C++ experience, then it means that the codebase isn't actually as bad as I think it is!

While it's true that the feature is quite easy to implement, I cannot accept a pull request on this as I still have my doubts about the image placement. Still, I wouldn't want to discourage anyone from contributing to Taiga or other open-source projects.

**Comment by akippnn** [comment]
I think it's better to have a preview panel instead.

This is an example design I made (Taiga-like application design prototype)
![image](https://user-images.githubusercontent.com/80670461/229645536-75f012cd-dcd8-4307-8aa5-842d3be3c1e7.png)

**Comment by koteq** [comment]
Rebasing my somewhat working solution on top of the current master branch, see https://github.com/koteq/taiga. It comes with a binary release, or GitHub Actions workflow for those who would like to build the binary themselves 🏗️

**Comment by amboris** [comment]
> Rebasing my somewhat working solution on top of the current master branch, see https://github.com/koteq/taiga. It comes with a binary release, or GitHub Actions workflow for those who would like to build the binary themselves 🏗️

![grafik](https://user-images.githubusercontent.com/16787537/230793827-3c2070d7-78c8-42f6-ba22-5f855c54d214.png)

I tried it out but for me it was bugged ^^"

**Comment by SacredZenpie** [comment]
@koteq Nice work! just on question, do you know why preview image randomly flickers when navigation the sidebar ui?

**Comment by hecode** [comment]
the exe gives TrojanDropper:Win32/Wacatac.B!ml

**Comment by SacredZenpie** [comment]
Are you sure its windows defender detecting it? Ive never had it false positive detect taiga before, but some antiviruses do.
